buildscripts/buildvars-setup.sh

```diff
@@ -149,7 +149,7 @@ export __UnprocessedBuildArgs=
 export __CleanBuild=0
 export __VerboseBuild=0
 export __ClangMajorVersion=3
-export __ClangMinorVersion=5
+export __ClangMinorVersion=8
```

Comment:
Should we update the required compiler version in a separate change instead mixing it with feature work?